s4e-backend/src/main/java/pl/cyfronet/s4e/bean/RegistrationToken.java

```diff
@@ -0,0 +1,24 @@
+package pl.cyfronet.s4e.bean;
+
+import lombok.Builder;
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@Builder
+public class RegistrationToken {
```

Comment:
not emailverification?

Reply:
I was considering EmailVerificationToken and it was too long for me. But actually just EmailVerification sounds better. The token is a component of such verification then. I'll refactor it.